### PR TITLE
Add more prominent feedback to Burn Wounds, as well as making Tajarans and Unathis less unique

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
All species that can get burn wounds now will get better feedback if they have received a burn wound/burn wounds, which includes the following:
a) a 65% chance to get a Runechat message about their charred flesh flaking off, and ;
b) a 35% chance to get a forced Hissing emote, which emits a sound cue and custom text

This PR also adds an additional *hiss emote for all human species, with an appropriate CC0 soundclip. Unathi and Tajaran's old *hiss soundclips are still perserved and now are unified under the general *hiss emote.

Note that during testing, the feedback seems to be too frequent to the point of being distracting, but I'm not sure if it is just me - If someone wishes for me to tune down the feedback frequency just tell me.

## Why It's Good For The Game
Burn wounds are notoriously bad for not giving feedback to the player while having an extremely debilitating drawback, which has spawned the meme of very successful traitors dropping dead suddenly because they didn't realise they got a dead heart due to an infection from a burn wound from a fight with Security 30 minutes ago.

While burn wounds now give you a giant warning message in the chatlog, people who are in the heat of combat may still miss it - Burn wounds giving an visual feedback in the game screen (Runechat) and an occasional auditory feedback (Forced *hiss emote) will make sure that the burn wound patient knows about the wound itself and seek treatment.

As a result of that, I had to add a separate *hiss soundclip for all other species, as well as unifying the two existing *hiss emote together. Tajarans and Unathis did lose their unique muzzled hiss text, but I consider that a worthy sacrifice.

N.B. Perhaps after this is in we could think about buffing burn wounds a bit - Burn wounds are notoriously easy to manage, but extremely debilitating when not noticed. Making it harder to get ignored may make burn wounds too easy to manage.

## Testing
Spawned a Tajaran, Unathi, and skrell. Inhabited and verified *hiss plays the correct soundclip and Runechat message.
Spawned a Vulpkanin and gave him an IK60, then inhabited them and shot at the three species to give them burn wounds. Then inhabited the three species and verified that the Runechat messages as well as the forced *hiss soundclips are correct.

## Changelog
:cl:
add: New *hiss emote for all human species
tweak: Tajaran and Unathi *hiss emotes now unified under general human *hiss emote
tweak: Burn wounds now have clearer feedback
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
> a) a 65% chance to get a Runechat message about their charred flesh flaking off, and ;

the rune chat brainrot.. I struggle to understand why burn wounds need to be unique with having runechat feedback, and NOTHING else has this. if people cant read chat, thats their fault

**Comment by SchrodingersWolf** [comment]
> > a) a 65% chance to get a Runechat message about their charred flesh flaking off, and ;
> 
> 
> 
> the rune chat brainrot.. I struggle to understand why burn wounds need to be unique with having runechat feedback, and NOTHING else has this. if people cant read chat, thats their fault

1080p, IB has that. You get the *coughs up blood message in both Runechat and the chatlog.

This merely emulates that behaviour.

**Comment by 1080pCat** [comment]
> > > a) a 65% chance to get a Runechat message about their charred flesh flaking off, and ;
> > 
> > 
> > the rune chat brainrot.. I struggle to understand why burn wounds need to be unique with having runechat feedback, and NOTHING else has this. if people cant read chat, thats their fault
> 
> 1080p, IB has that. You get the *coughs up blood message in both Runechat and the chatlog.
> 
> This merely emulates that behaviour.

Yes, but IB makes you emote, rather than just something put into your chat

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Scribble-Sheep** [comment]
This is...actually gonna mess with me lol.
I have *hiss/*aflap/*quill bound to my harm intent button bc they're mutually exclusive.
Not anymore tho :<

File: code/datums/keybindings/emote_keybinds.dm [hunk 1]
```diff
@@ -502,8 +502,8 @@
 	name = "Rumble"
 
 /datum/keybinding/emote/carbon/human/hiss
-	linked_emote = /datum/emote/living/carbon/human/hiss
-	name = "Hiss (Unathi)"
+	linked_emote = /datum/emote/living/carbon/human/hiss/
```
Comment (lewcc) [inline code/datums/keybindings/emote_keybinds.dm:505]:
```suggestion
	linked_emote = /datum/emote/living/carbon/human/hiss
```

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 2]
```diff
@@ -31,6 +31,31 @@
...
+	message_param = "hisses at %t."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+	muzzled_noises = list("weak hissing", "irritated")
+
+/datum/emote/living/carbon/human/hiss/get_sound(mob/living/user)
+	var/mob/living/carbon/human/H = user
+	if(H.mind?.miming || !istype(H))
+		return
+	else
+		return H.dna.species.hiss_sound
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:48]: Slight reorganization here
```suggestion
	if(!istype(H) || H.mind?.miming)
		return
	return H.dna.species.hiss_sound
```

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 3]
```diff
@@ -31,6 +31,31 @@
...
+
+/datum/emote/living/carbon/human/hiss/select_message_type(mob/user, msg, intentional) //Only used when emote forced by burn wounds
+	. = ..()
+	var/mob/living/carbon/human/H = user
+	if(intentional == FALSE)
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:53]:
```suggestion
	if(!intentional)
```

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 4]
```diff
@@ -31,6 +31,31 @@
...
+		if(H.mind?.miming)
+			return "grimaces as their skin peels off!"
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:55]: Honestly, I think you might want to make a subtype of hiss with a slightly different key that gets called for this. Something like burn_hiss. I think it's entirely reasonable that we might want hiss to get called unintentionally without giving the flavor that their flesh is peeling off.

**Review by lewcc — CHANGES_REQUESTED** [review]

**Comment by SchrodingersWolf** [comment]
I screwed up with my branch - Sorry for all this mess

File: code/modules/mob/living/carbon/human/human_emote.dm (hunk 4 above)
Comment (SchrodingersWolf) [reply code/modules/mob/living/carbon/human/human_emote.dm:55]: Good point - I originally wanted to make it a subtype but just not sure if that would bloat the file too much.

Comment (lewcc) [reply code/modules/mob/living/carbon/human/human_emote.dm:55]: Code quality is much more important than worrying about file bloat, especially in a file that's already as big as this

Comment (SchrodingersWolf) [reply code/modules/mob/living/carbon/human/human_emote.dm]: Changes now in newest commit

File: code/modules/mob/living/carbon/human/human_emote.dm (hunk 3 above)
Comment (SchrodingersWolf) [reply code/modules/mob/living/carbon/human/human_emote.dm]: Check is now obsolete in newest commit.

File: code/modules/mob/living/carbon/human/human_emote.dm (hunk 2 above)
Comment (SchrodingersWolf) [reply code/modules/mob/living/carbon/human/human_emote.dm]: Applied suggestion in newest commit

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 5]
```diff
@@ -31,6 +31,34 @@
...
+	key_third_person = "burn_hisses"
+	message = "hisses as their skin peels off!"
+	message_mime = "grimaces as their skin peels off!."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+	muzzled_noises = list("pained hissing")
+
+/datum/emote/living/carbon/human/burn_hiss/run_emote(mob/user, params, type_override, intentional)
+	var/mob/living/carbon/human/H = user
+	if(!istype(H) || !intentional)
+		return
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:60]: Is this doing anything? I think you might need to call parent here

Comment (SchrodingersWolf) [reply code/modules/mob/living/carbon/human/human_emote.dm:60]: During testing my skrell hisses normally, so I don't know?
I can just call parent here as well

Comment (lewcc) [reply code/modules/mob/living/carbon/human/human_emote.dm:61]: I think what you *want* is to check the condition before making the parent call. If the condition is false, just return FALSE, and after that conditional `return ..()`. That make sense?

**Comment by Fordoxia** [comment]
The runechat message, I am fine with. The forced *hiss emote I am not fine with.

Forced audible emote will mean that you can be detected from beyond line of sight. You can be heard hissing from inside a hidey hole whilst trying to break off from sec so you can go and heal.

**Review by Burzah — CHANGES_REQUESTED** [review]
Please put credit to the clips used for this in the PR description itself.

**Review by FunnyMan3595 — APPROVED** [review]

**Pull request closed without merging**